### DISCUSSION TITLE
Client Defined Multiplier of Power

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/FIPs#1083] [Enhancements - Technical]
## Abstract

Allow storage clients to define integer power multiplier from 0x to 10x when making storage deals.

## Motivation

This proposal intends to rejuvenate the network from stagnation by providing a way for storage client to express their risk preferences through custom multiplier values, which we think lays the foundation for the ultimate storage market where any particular pair of client and provider could dictate their own storage terms with regards to pledge, penalty and rewards. We believe implementing such feature could have following positive effects on the network...

### Network growth

This proposal could potentially rekindle a new wave of network growth driven by storage clients creating storage demand for more SPs to come back to the network and to provide wealth of storage supply to network to meet such demand. As network growth has strong implications on network security and market perceptions of the network, it is our belief that client multiplier could have an optimistic impact on the network as a whole.

### Price discovery

By allowing more flexible market expressions from storage clients, storage market can then be gamed by both clients and providers to eventually reach equilibrium that may reflect whatever realities (regulation, financial situation or geolocation issues) a particular pair of client and provider are involved in, with paid deals reached between them. 

### Future programability 

Client multiplier would also be the foundational primitive which FEVM could take advantage of for a future of contract based storage market to allow the ultimate storage market to be conceived on top of the network.

_A proposal by [Ancients Research](https://ancientsresearch.xyz/) in an effort to MFGA (Make Filecoin Great Again!)_

<img src="https://github.com/user-attachments/assets/c5ecfcc2-5ff1-4793-9ddf-3d5cb36de5e0" width="24">

**Comment by 15012700225** [comment]
Good Idea. Support.

**Comment by anorth** [comment]
Thanks for this simple proposal @Fatman13.  I'll point out for readers that this would have largely the same effect as https://github.com/filecoin-project/FIPs/discussions/844, with a much simpler overall system (but slightly more effort to get there). The resulting simplicity would lead me to prefer this version to the earlier one.

**Comment by hannahhoward** [comment]
question: why wouldn't a CC miner just create a fake client and give themselves a super high multiplier on a bunch of deals for their own CC sectors?

just want to understand.

**Reply by anorth** [reply]
They don't need a fake client or any deals. They can just give themselves a high (10x bound) multiplier on their CC sector.

Why wouldn't they? Hypothetically they are capital constrained, or don't want to take that risk, or don't care about block rewards b/c they're doing service for a paying client. Probably all three at once.

**Reply by Fatman13** [reply]
i personally think the proposal as a **paradigm shift** towards an environment that protocol / tooling is somewhat impartial to client provider relationships. Similar questions could also be formed against current data-onboarding paradigm, but really the core question is which paradigm we believe we can reach consensus on to solve a [crisis](https://filecoinproject.slack.com/archives/C047LKMDRLM/p1733739047430229). 

> History of science indicates that, particularly in the early developmental stages of a new paradigm, it is not even very difficult to invent such alternates. But that invention of alternates is just what scientists seldom undertake except during the pre-paradigm stage of their science’s development and at very special occasions during its subsequent evolution. So long as the tools a paradigm supplies continue to prove capable of solving the problems it defines, science moves fastest and penetrates most deeply through confident employment of those tools. The reason is clear. As in manufacture so in science—retooling is an extravagance to be reserved for the occasion that demands it. The significance of crises is the indication they provide that an occasion for retooling has arrived. 
> 
> -- [The Structure of Scientific Revolutions](https://www.lri.fr/~mbl/Stanford/CS477/papers/Kuhn-SSR-2ndEd.pdf), by Thomas Kuhn

**Comment by mjroddy** [comment]
Thanks @Fatman13 

So I understand properly....

If anyone can provide their own pledge, penalty and rewards.

Would this not impact security of network or are we saying within network minimums or something?

**Reply by Fatman13** [reply]
> If anyone can provide their own pledge, penalty and rewards.

That would be the ultimate goal for a contract based storage market on L2, akin to what is described [here](https://github.com/filecoin-project/FIPs/discussions/725) in the master plan.

> Would this not impact security of network or are we saying within network minimums or something?

Network security is still maintained by the hardware + stake structure unchanged, with potentials to make core protocol [simpler](https://github.com/filecoin-project/FIPs/discussions/1036) in the future.

**Comment by sviperj** [comment]
Good Idea. Support.

**Reply by Fatman13** [reply]
兄弟有没有兴趣参加 [asia spwg](https://filecoinproject.slack.com/archives/CR98WERRN/p1756361532889529)？
